src/main/java/org/codehaus/mojo/flatten/AbstractFlattenMojo.java

```diff
@@ -43,6 +43,12 @@ public abstract class AbstractFlattenMojo extends AbstractMojo {
     @Parameter(property = "flattenedPomFilename", defaultValue = ".flattened-pom.xml")
     private String flattenedPomFilename;
 
+    /**
+     * If {@code true} the plugin will be skipped.
+     */
+    @Parameter(property = "flatten.skip", defaultValue = "false")
```

Comment:
Please add `@since` tag, as it is a new feature we will have a 1.6.0 version.
Please add for all new parameters.

You can also update project version to 1.6.0-SNAPSHOT to not a miss during next release.

Reply:
@slawekjaranowski `@since` tag added and project version updated